csrc/scheduler/reduction_heuristic.h

```diff
@@ -125,6 +125,7 @@ class ReductionParams : public HeuristicParams {
   bool tidx_for_outer_reduction = false;
   // pad outer reduction to warp
   bool pad_outer_reduction_to_warp = false;
+  bool combined_split_grid_inner_dim = false;
```

Comment:
nitpick: let's put a comment here on what this field does and why.

Reply:
added